test/ReplicatorCollectionTest.cc

```diff
@@ -624,6 +624,69 @@ TEST_CASE_METHOD(ReplicatorCollectionTest, "Conflict Resolver with Collections",
     CBLDocument_Release(bar1);
 }
 
+TEST_CASE_METHOD(ReplicatorCollectionTest, "Resolve Pending Conflicts", "[Replicator]") {
+    CBLLog_SetConsoleLevel(kCBLLogVerbose);
```

Comment:
You should probably not keep this here as it will stick through the rest of the tests

Reply:
Thanks. Having there for debugging so removing it.